aiohttp/server.py

```diff
@@ -103,6 +106,14 @@ def __init__(self, *, loop=None,
         self._timeout = timeout  # slow request timeout
         self._loop = loop if loop is not None else asyncio.get_event_loop()
 
+        parser_kwargs = {}
```

Comment:
I wanted to avoid giving default values for those parameters away from the
place where the parameters are used.  That is, if I was looking for the
default value of max_headers, I would go looking in the parser class, not
here.  Is there a more py3-ish way to accomplish that?